tests/bend_tests.ts

```diff
@@ -265,4 +248,11 @@ const BendTests = {
     ok(true, 'Whako Release');
   },
 };
+
+//#region Helper Functions
+const note = (tab_struct: TabNoteStruct) => new TabNote(tab_struct);
+const bendWithText = (text: string, release = false) => new Bend(text, release);
+const bendWithPhrase = (phrase: BendPhrase[]) => new Bend('', false, phrase);
+//#endregion Helper Functions
```

Comment:
What are these `#region/endregion` annotations?

Also, can we move these helpers to the top? It's more readable if dependencies are declared before the they are used.

Reply:
https://github.com/0xfe/vexflow/pull/1094#issuecomment-908785984

The above comment discusses my overall approach to cleaning up the tests.

#region markers are VS code specific and enable large blocks of code to be folded with a shortcut key (e.g. ctrl+shift+[ on windows / cmd+opt+[ on mac) or by clicking on the left gutter. I have started using them in my recent PRs for large related blocks of code. 

For example in a later PR, I noticed that glyph.ts has 180 lines of static methods before the constructor(). I plan to add #region / #endregion blocks around some of these larger blocks to assist developers with navigating around the code.

https://github.com/0xfe/vexflow/blob/29a838e5d7a940c4b6c74afb2d19741d4ea67fab/src/glyph.ts#L186-190

That said...some of these helper function regions aren't that long. I put them everywhere for consistency's sake, but maybe they should only appear if the region is very long and unwieldy, like over 150-200 lines of code.

Also, they are specific to a particular IDE and are a little ugly (albeit useful!).

If you prefer to not have the #region markers, I can totally remove them.

Reply:
Thanks Ron. I'm not really a fan of the syntax, or the vs-code specific behavior. If we're going to use them, then let's use them sparingly where there's large blocks of clear noise.

Reply:
Agreed! I also think the syntax looks weird, but it seemed useful while I was extracting some of the helper functions.

I'll remove the #region annotations.